apps/web/frontend/components/hourInput.tsx

```diff
@@ -64,6 +64,7 @@ export const HourInput = (props: {
   className?: string
   onBlur?: (workHours: number) => void
   readOnly?: boolean
+  isdisabled?: boolean
```

Comment:
Please use camelCase, so either `disabled` or `isDisabeld`.
I would prefer `disabled`, as the attribute on an HTML input is also named `disabled`.